### DISCUSSION TITLE
blog/exploring-the-shells

### DISCUSSION THREAD

**Discussion by giscus** [linux-club-dev/linux-club-web#5] [General]
# blog/exploring-the-shells

Discover the world of shells through this beginner-friendly blog.

http://localhost:3000/blog/exploring-the-shells

**Comment by SAYUK09** [comment]
Great Blog

**Comment by vrushalikudande** [comment]
👍

**Comment by Arnavpk** [comment]
It seems very interesting, got to learn more about zsh & fish. Looking ahead to read more blogs of you.
